app/src/main/java/com/nextcloud/talk/diagnose/DiagnoseActivity.kt

```diff
@@ -187,6 +196,10 @@ class DiagnoseActivity : BaseActivity() {
         ).show()
     }
 
+    private fun testPushNotification() {
+        addHeadline(context.resources.getString(R.string.nc_test_push_button))
```

Comment:
by adding this via addHeadline the string will be included in diagnoseData, so it will be contained as headline in the copied markdown which doesn't make sense.

Then also the check
`if (element.headline == "Test push notifications") {`
in DiagnoseContentComposable is not necessary. 
Just combine it with ShowNotificationData in DiagnoseContentComposable to initially show the Button.